### PR TITLE
Phishing enhancements spear

### PR DESCRIPTION
**Pull request by ArikDay**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-5630)

## Description
Playbook - 
- Analyst can now input a list of keywords to search in the phishing email body using the new input "KeyWordsToSearch"
- A new sub-playbook added - "Spear Phishing Investigation"
- DomainSquatting capability moved from the main playbook to the new sub-playbook
Layout
- "Threat Intelligence Analysis" section has been added to the "Investigation" tab
- New section added - "Spear Phishing Investigation"
- DomainSquatting incident field has moved to the new section

## Screenshots
Phishing_-_Generic_v3_6_5
Spear_Phishing_Investigation
## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Phishing/Playbooks/Phishing_-_Generic_v3_6_5_README.md [hunk 1]
```diff
@@ -75,8 +73,9 @@ This playbook does not use any integrations.
...
-| TakeManualActions | Specify whether to stop the playbook to take additional action before closing the incident.<br/>Set the value to 'True' to stop the playbook before closing the incidents, or "False" to close the incident once the playbook flow is done. | false | Optional |
+| UserEngagement | Specify whether to engage with the user via email for investigation updates.<br/>Set the value to 'True' to allow user engagement, or 'False' to avoid user engagement. | True | Optional |
+| TakeManualActions | Specify whether to stop the playbook to take additional action before closing the incident.<br/>Set the value to 'True' to stop the playbook before closing the incidents, or "False" to close the incident once the playbook flow is done. | False | Optional |
+| KeyWordsToSearch | A comma-separated or list of keywords to search in the email body.<br/>for example: name of the organization finance app that the attacker might impost to.<br/>This input is being used in the "Spear Phishing Investigation" sub-playbook. |  | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/Playbooks/Phishing_-_Generic_v3_6_5_README.md:78]:
```suggestion
| KeyWordsToSearch | A comma-separated or list of keywords to search in the email body.<br/>For example: name of the organization finance app that the attacker might impersonate.<br/>This input is being used in the "Spear Phishing Investigation" sub-playbook. |  | Optional |
```

File: Packs/Phishing/Playbooks/playbook-Spear_Phishing_Investigation.yml [hunk 2]
```diff
@@ -0,0 +1,437 @@
+id: Spear Phishing Investigation
+version: -1
+name: Spear Phishing Investigation
+description: The "Spear Phishing Investigation" Playbook designed to detect patterns that indicates a spear phishing attempt by the attacker.
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/Playbooks/playbook-Spear_Phishing_Investigation.yml:4]:
```suggestion
description: The "Spear Phishing Investigation" playbook is designed to detect patterns that indicates a spear phishing attempt by the attacker.
```

File: Packs/Phishing/Playbooks/playbook-Spear_Phishing_Investigation.yml [hunk 3]
```diff
@@ -0,0 +1,437 @@
...
+        "x": -80,
+        "y": 50
+      }
+    }
+  }
+inputs:
+- key: DomainSquattingResults
+  value: {}
+  required: false
+  description: Results of the domain squatting investigation
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/Playbooks/playbook-Spear_Phishing_Investigation.yml:420]:
```suggestion
  description: Results of the domain squatting investigation.
```

File: Packs/Phishing/Playbooks/playbook-Spear_Phishing_Investigation.yml [hunk 4]
```diff
@@ -0,0 +1,437 @@
...
+  playbookInputQuery:
+- key: KeywordsToSearch
+  value: {}
+  required: false
+  description: |-
+    A comma-separated or list of keywords to search in the email body.
+    for example: name of the organization finance app that the attacker might impost to.
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/Playbooks/playbook-Spear_Phishing_Investigation.yml:427]:
```suggestion
    For example: name of the organization finance app that the attacker might impersonate.
```

File: Packs/Phishing/Playbooks/playbook-Spear_Phishing_Investigation_README.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+The "Spear Phishing Investigation" Playbook designed to detect patterns that indicates a spear phishing attempt by the attacker.
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/Playbooks/playbook-Spear_Phishing_Investigation_README.md:1]:
```suggestion
The "Spear Phishing Investigation" playbook is designed to detect patterns that indicates a spear phishing attempt by the attacker.
```

File: Packs/Phishing/Playbooks/playbook-Spear_Phishing_Investigation_README.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+This playbook does not use any sub-playbooks.
+
+### Integrations
+
+This playbook does not use any integrations.
+
+### Scripts
+
+* MatchRegexV2
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/Playbooks/playbook-Spear_Phishing_Investigation_README.md:17]:
```suggestion
MatchRegexV2
```

File: Packs/Phishing/Playbooks/playbook-Spear_Phishing_Investigation_README.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+### Commands
+
+* setIncident
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/Playbooks/playbook-Spear_Phishing_Investigation_README.md:21]:
```suggestion
setIncident
```

File: Packs/Phishing/Playbooks/playbook-Spear_Phishing_Investigation_README.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+## Playbook Inputs
+
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| DomainSquattingResults | Results of the domain squatting investigation |  | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/Playbooks/playbook-Spear_Phishing_Investigation_README.md:29]:
```suggestion
| DomainSquattingResults | Results of the domain squatting investigation. |  | Optional |
```

File: Packs/Phishing/Playbooks/playbook-Spear_Phishing_Investigation_README.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+| KeywordsToSearch | A comma-separated or list of keywords to search in the email body.<br/>for example: name of the organization finance app that the attacker might impost to. |  | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/Playbooks/playbook-Spear_Phishing_Investigation_README.md:30]:
```suggestion
| KeywordsToSearch | A comma-separated list of keywords to search in the email body.<br/>For example: name of the organization finance app that the attacker might impersonate. |  | Optional |
```

File: Packs/Phishing/Playbooks/playbook-Spear_Phishing_Investigation.yml [hunk 10]
```diff
@@ -0,0 +1,437 @@
...
+- key: DomainSquattingResults
+  value: {}
+  required: false
+  description: Results of the domain squatting investigation
+  playbookInputQuery:
+- key: KeywordsToSearch
+  value: {}
+  required: false
+  description: |-
+    A comma-separated or list of keywords to search in the email body.
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/Playbooks/playbook-Spear_Phishing_Investigation.yml:426]:
```suggestion
    A comma-separated list of keywords to search in the email body.
```

File: Packs/Phishing/Playbooks/Phishing_-_Generic_v3_6_5.yml [hunk 11]
```diff
@@ -3741,6 +3662,14 @@ inputs:
     Specify whether to stop the playbook to take additional action before closing the incident.
     Set the value to 'True' to stop the playbook before closing the incidents, or "False" to close the incident once the playbook flow is done.
   playbookInputQuery:
+- key: KeyWordsToSearch
+  value: {}
+  required: false
+  description: |-
+    A comma-separated or list of keywords to search in the email body.
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/Playbooks/Phishing_-_Generic_v3_6_5.yml:3669]:
```suggestion
    A comma-separated list of keywords to search in the email body.
```

File: Packs/Phishing/ReleaseNotes/3_5_20.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
+
+#### Playbooks
+
+##### Phishing - Generic v3
+
+- Analyst can now input a list of keywords to search in the phishing email body using the new input "KeyWordsToSearch"
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/ReleaseNotes/3_5_20.md:6]:
```suggestion
- Analyst can now input a list of keywords to search in the phishing email body using the new input "KeyWordsToSearch".
```

File: Packs/Phishing/ReleaseNotes/3_5_20.md [hunk 13]
```diff
@@ -0,0 +1,27 @@
...
+- A new sub-playbook added - "Spear Phishing Investigation"
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/ReleaseNotes/3_5_20.md:7]:
```suggestion
- A new sub-playbook added - "Spear Phishing Investigation".
```

File: Packs/Phishing/ReleaseNotes/3_5_20.md [hunk 14]
```diff
@@ -0,0 +1,27 @@
...
+- DomainSquatting capability moved from the main playbook to the new sub-playbook
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/ReleaseNotes/3_5_20.md:8]:
```suggestion
- DomainSquatting capability moved from the main playbook to the new sub-playbook.
```

File: Packs/Phishing/ReleaseNotes/3_5_20.md [hunk 15]
```diff
@@ -0,0 +1,27 @@
...
+
+##### New: Spear Phishing Investigation
+
+- New: The "Spear Phishing Investigation" Playbook designed to detect patterns that indicates a spear phishing attempt by the attacker.
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/ReleaseNotes/3_5_20.md:12]:
```suggestion
New: The "Spear Phishing Investigation" playbook is designed to detect patterns that indicates a spear phishing attempt by the attacker.
```

File: Packs/Phishing/ReleaseNotes/3_5_20.md [hunk 16]
```diff
@@ -0,0 +1,27 @@
...
+
+#### Layouts
+
+##### Phishing Incident v3
+
+- "Threat Intelligence Analysis" section has been added to the "Investigation" tab
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/ReleaseNotes/3_5_20.md:18]:
```suggestion
- "Threat Intelligence Analysis" section has been added to the "Investigation" tab.
```

File: Packs/Phishing/ReleaseNotes/3_5_20.md [hunk 17]
```diff
@@ -0,0 +1,27 @@
...
+- New section added - "Spear Phishing Investigation"
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/ReleaseNotes/3_5_20.md:19]:
```suggestion
- New section added - "Spear Phishing Investigation".
```

File: Packs/Phishing/ReleaseNotes/3_5_20.md [hunk 18]
```diff
@@ -0,0 +1,27 @@
...
+- DomainSquatting incident field has moved to the new section
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/ReleaseNotes/3_5_20.md:20]:
```suggestion
- DomainSquatting incident field has moved to the new section.
```

File: Packs/Phishing/ReleaseNotes/3_5_20.md [hunk 19]
```diff
@@ -0,0 +1,27 @@
...
+
+
+
+#### Incident Fields
+
+- New: **Email Keywords Found**
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/ReleaseNotes/3_5_20.md:26]:
```suggestion
.New: **Email Keywords Found**
```

**Comment by ShirleyDenkberg** [comment]
@tomer-pan @AdiPeret Doc review completed.

**Review by tomer-pan — APPROVED** [review]
Looks awesome, great job!

**Pull request merged**